index.js

```diff
@@ -3,6 +3,10 @@ const morgan = require('morgan');
 
 const app = express();
 
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
Do we need this?

Reply:
Really depends on what HTTP request module they want to use on the front end, for now URL encoded requests and JSON requests for testing via postman perhaps are enabled!